index.html

```diff
@@ -233,6 +233,47 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
```

Comment:
- [ ] Like the previous spoke about this part, kindly align this particular item to the center like it is in the Figma design, to enable a pixel-perfect design 😀.